src/main/java/org/sopt36/ninedotserver/mandalart/controller/SubGoalController.java

```diff
@@ -1,15 +1,37 @@
 package org.sopt36.ninedotserver.mandalart.controller;
 
+import java.util.List;
+import java.util.Map;
 import lombok.RequiredArgsConstructor;
+import org.sopt36.ninedotserver.global.dto.response.ApiResponse;
+import org.sopt36.ninedotserver.mandalart.dto.response.SubGoalIdResponse;
 import org.sopt36.ninedotserver.mandalart.service.command.SubGoalCommandService;
 import org.sopt36.ninedotserver.mandalart.service.query.SubGoalQueryService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RequiredArgsConstructor
 @RestController
+@RequestMapping("/api/v1/core-goals")
 public class SubGoalController {
 
     private final SubGoalCommandService subGoalCommandService;
     private final SubGoalQueryService subGoalQueryService;
 
+    @GetMapping("/{coreGoalId}/sub-goals")
+    public ResponseEntity<ApiResponse<Map<String, List<SubGoalIdResponse>>, Void>> getSubGoalIds(
+        @PathVariable Long coreGoalId
+    ) {
+        Long userId = 1L; // TODO: 로그인 구현되면 token에서 사용자id 가져오기
+        List<SubGoalIdResponse> subGoalIds = subGoalQueryService.getSubGoalIds(userId, coreGoalId);
+        Map<String, List<SubGoalIdResponse>> response = Map.of("subGoalIds", subGoalIds);
```

Comment:
요거 List<SubGoalIdResponse>를 Dto단으로 한 번 더 묶어서 해당 Dto 반환받는 방식으로 코딩하면 더욱 좋을 것 같습니다!
```suggestion
public SubGoalIdsDto getSubGoalIds() {
    List<SubGoalIdResponse> subGoalIds = ...; // 기존 로직
    return new SubGoalIdsDto(subGoalIds);
}

// 해당 부분 호출 로직
SubGoalIdsDto dto = new SubGoalIdsDto(subGoalIds);

```